scripts/bundleSize.js

```diff
@@ -0,0 +1,145 @@
+#! /usr/bin/env node
+
+const ora = require('ora');
+const fs = require('fs');
+const chalk = require('chalk');
+
+const { MIN_SIZE, MAX_SIZE, MIN_SERVICES } = require('./bundleSizeConfig');
+
+const jsFiles = fs
+  .readdirSync('build/public/static/js')
+  .filter(fileName => fileName.endsWith('.js'));
+
+const services = jsFiles
```

Comment:
Hey, soz for the flip flop, weve actually now got an cjs way to get all services, Object.keys(services) from `simorgh/cypress/support/config/services.js` should do the tricky :) If your too busy to update it no worries, just say and I can make the change :)

doing this should mean `MIN_SERVICES` shouldnt be needed, after which LGTM :)